gatsby-node.js

```diff
@@ -219,6 +219,50 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     }
   });
 
+  const createEmbed = (node) => {
+    const {
+      frontmatter: { redirects },
+      fields: { fileRelativePath, slug },
+    } = node;
+
+    if (
+      fileRelativePath.includes('src/content/docs/release-notes') ||
+      fileRelativePath.includes('src/content/whats-new')
+    ) {
+      return;
+    }
+
+    const pagePath = path.join(slug, 'embed', '/');
+
+    createPage({
+      path: pagePath,
+      component: path.resolve(`src/templates/embedPage.js`),
+      context: {
+        slug,
+        fileRelativePath,
+        layout: 'EmbedLayout',
+      },
+    });
+
+    if (redirects) {
+      redirects.forEach((fromPath) => {
+        reporter.info(
+          `Creating redirect for embed page: ${path.join(
+            fromPath,
+            'embed',
+            '/'
+          )}`
+        );
+        createRedirect({
+          fromPath: path.join(fromPath, 'embed', '/'),
+          toPath: pagePath,
+          isPermanent: true,
+          redirectInBrowser: true,
+        });
+      });
```

Comment:
do we need to create redirects for embed pages also? 

if when DLS gets a result and checks for redirect on the standard page first, THEN gets the new URL and appends `/embed/` onto the end, i don't think we need redirects right?

Reply:
if we see that the only use for the `/embed/` is to be used for the DLS and no other potential use cases in the future, then we can remove this

Reply:
for now, yes, we only need to support embed pages via DLS. and we can document that. best spot i think is likely our site architecture wiki page